### ISSUE TITLE
Processed data lacks files

### ISSUE THREAD

**Issue by gofortargets** [nlpyang/BertSum#52]
Dear author, 

May I ask why the number of data points in processed training data is only 287084, while the paper claims it uses 287227 training points? 

Looking forward to your reply, 
Thank you,

**Comment by nlpyang** [comment]
We removed some documents which are too short.

**Comment by gofortargets** [comment]
Also the same reason is applied to dev and test set ? Thanks

**Comment by nlpyang** [comment]
Yes, you could see preprocess scritps for details. But for test set, no documents were removed.
